### PR TITLE
Issue 29. Add basic authentication quarantine

### LINKED ISSUE #29: Add quarantine for a client not passing the authentication/authorization
The quarantine time must be configured with the configuration file parameter.

### PR DESCRIPTION
**Pull request by ksenia-vazhdaeva**

### CODE REVIEW

**Comment by ksenia-vazhdaeva** [comment]
Based on https://github.com/insight-platform/MediaGateway/pull/34

File: docs/source/cookbook/0_https.rst [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+Server
+""""""
+
+To test the server only a certificate with IP SAN is used.
+
+Prepare the configuration file
+
+.. code-block:: bash
+
+    echo "{
```
Comment (bwsw) [inline docs/source/cookbook/0_https.rst]: Please replace with here docs everything with massive backslash jsons/yamls.

File: docs/source/cookbook/2_basic_auth.rst [hunk 2]
```diff
@@ -39,37 +125,32 @@ To generate certificates signed by a private CA
...
     openssl req -new -key certs/client.key -out certs/client.csr -subj "/CN=etcd-client"
 
     # Generate client certificate signed by the CA
     openssl x509 -req -days 365 -in certs/client.csr -CA certs/ca.crt -CAkey certs/ca.key -CAcreateserial -out certs/client.crt
 
-    chmod 644 certs/ca.key certs/server.key certs/client.key
-
-``certs/*.crt`` is a file with a certificate in PEM format and ``certs/*.key`` is a file with a key in PEM format where ``*`` is a replacement for `ca`, `server` and `client`.
-
 Launching etcd
```
Comment (bwsw) [inline docs/source/cookbook/2_basic_auth.rst:143]: Change to Etcd (capital letter). Check other places too.

File: docs/source/cookbook/2_basic_auth.rst [hunk 3]
```diff
@@ -241,10 +321,6 @@ The configuration below does not contain TLS settings for simplicity. For produc
                     \"urls\": [
```
Comment (bwsw) [inline docs/source/cookbook/2_basic_auth.rst]: here-doc

**Review by bwsw — CHANGES_REQUESTED** [review]

File: docs/source/cookbook/2_basic_auth.rst (hunk 2 above)
Comment (ksenia-vazhdaeva) [reply docs/source/cookbook/2_basic_auth.rst:143]: I used the same writing as in the official documentation - `etcd`. See https://etcd.io/

File: docs/source/cookbook/0_https.rst (hunk 1 above)
Comment (ksenia-vazhdaeva) [reply docs/source/cookbook/0_https.rst]: What is meant by here docs?

File: docs/source/cookbook/2_basic_auth.rst (hunk 3 above)
Comment (ksenia-vazhdaeva) [reply docs/source/cookbook/2_basic_auth.rst]: Fixed in https://github.com/insight-platform/MediaGateway/pull/34

File: docs/source/cookbook/0_https.rst (hunk 1 above)
Comment (ksenia-vazhdaeva) [reply docs/source/cookbook/0_https.rst]: Fixed in https://github.com/insight-platform/MediaGateway/pull/34

**Review by bwsw — APPROVED** [review]

**Pull request merged**